vault/logical_system_quotas.go

```diff
@@ -73,6 +73,11 @@ The 'rate' must be positive.`,
 					Type:        framework.TypeDurationSecond,
 					Description: "The duration to enforce rate limiting for (default '1s').",
 				},
+				"block": {
```

Comment:
This is feeling like a boolean knob. How about `block_interval`? That would indicate that it is a duration and consistent with the other param `interval`.